### ISSUE TITLE
Add an option to speak player facing direction in coordinate terms instead of cardinal directions

### ISSUE THREAD

**Issue by Molitvan** [minecraft-access/minecraft-access#387]
When enabled, this option will replace all instances of cardinal directions (north, east etc) with their respective coordinate direction counterparts (negative z, positive x etc). Up and down will stay the same because I don't think there's ever a need for repeating that up is positive y and down is negative y.

**Comment by francescoloiacono** [comment]
Hi nice to meet you,
can i contribute to this issue?

**Comment by Molitvan** [comment]
> Hi nice to meet you,
> can i contribute to this issue?

Of course. Any contribution is welcome. If you think you can add this, do it and open a pull request. Also thanks for the interest 🙂
